test/state/state.hpp

```diff
@@ -14,55 +14,55 @@
 
 namespace evmone::state
 {
-struct JournalBase
+/// The Ethereum State: the collection of accounts mapped by their addresses.
+///
+/// TODO: This class is copyable for testing. Consider making it non-copyable.
+class State
 {
-    address addr;
-};
+    struct JournalBase
```

Comment:
Would be good to have some doc comments for base and deriving structs, but I can live w/o it.

Reply:
This is going to be updated later.